### DISCUSSION TITLE
Add Figma Profile Iconography

### DISCUSSION THREAD

**Discussion by JamesIves** [community/community#130409] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

It would be great to add shortlinks and iconography for Figma profiles on the GitHub profile.

For example when you add a LinkedIn url, it shortens it to `in/namehere` and displays a LinkedIn icon. I'd love to see to see the same for Figma profiles. There's a lot of overlap between the Figma community and plugins in the sense that these are shared resources, plus having iconography would make profiles look pretty.

**Comment by Rishi-Sudhakar** [comment]
You could add the link, But it will only show the "whole url". It works with twitter, Reddit, linkedin, instagram etc though. But yes figma would be nice.

**Comment by ebndev** [comment]
Hi @JamesIves, thanks for being a part of the GitHub Community!

We sincerely appreciate you taking the time to share your thoughts.

As a valued GitHub user, your feedback in this Discussion space, along with appropriate tags and examples, plays a crucial role in shaping the development of GitHub. While we can’t guarantee immediate action on every piece of feedback, please know that we are attentive to your suggestions and continuously strive to enhance the GitHub experience.

**Comment by pjpscriv** [comment]
@JamesIves I've created a little Chrome extension as a temporary fix for this, check it out here: https://chromewebstore.google.com/detail/github-social-icons/ledidlccnpfooopagmkgihgibfbnbbnp 😊

![image](https://github.com/user-attachments/assets/ea9902d7-eebe-4a76-be6f-401da76e17b6)
